core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -750,32 +750,32 @@ object Parser {
         private val dq = "\""
         def show(expectation: Expectation): String = expectation match {
           case OneOfStr(_, strs: List[String]) =>
-            "one of: " + strs.map(s => dq + s + dq).mkString("{", ", ", "}")
+            "must match one of the strings: " + strs.map(s => dq + s + dq).mkString("{", ", ", "}")
```

Comment:
nit: `strs.iterator.map(..).mkString` will be faster and avoid allocating an intermediate list.